src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+    public String getDescription() {
+        return description;
+    }
+    @Override
+public String toString()
+
+{
+    return getStatusIcon() + " " + getDescription();
+
+    }
+    public String taskDescription() {
+        return  getStatusIcon() + " " + getDescription();
+    }
+    public String getStatusIcon() {
+        return (isDone ? "[X]" : "[ ]"); // mark done task with X
+    }
+    public Boolean changeStatus() {
+        return this.isDone;
+    }
+
+
+public void setDone() {
+        this.isDone = true;
+}
+ public void setNotDone() {
+        this.isDone = false;
+ }   //...
```

Comment:
Please remove unnecessary comment if not needed, this applies to other parts of the program too